src/index.html

```diff
@@ -3,10 +3,381 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
+  <header class="header">
+    <a target="_blank" href="#home">
+      <img
+        class="logo"
+        src="images/logo.png"
+        alt="Moyo logo"
+      >
+    </a>
+    <nav class="nav">
+      <ul class="nav__list">
+        <li class="nav__item">
+          <a href="#apple" class="nav__link is-active">apple</a>
+        </li>
+        <li class="nav__item">
+          <a href="#samsung"class="nav__link">samsung</a>
+        </li>
+        <li class="nav__item">
+          <a href="#smartphones"class="nav__link">smartphones</a>
+        </li>
+        <li class="nav__item">
+          <a
+            href="#laptops&computers"
+            class="nav__link"
+            data-qa="nav-hover"
+          >
+          laptops & computers
+          </a>
+        </li>
+        <li class="nav__item">
+          <a href="#gadgets"class="nav__link">gadgets</a>
+        </li>
+        <li class="nav__item">
+          <a href="#tablets"class="nav__link">tablets</a>
+        </li>
+        <li class="nav__item">
+          <a href="#photo"class="nav__link">photo</a>
+        </li>
+        <li class="nav__item">
+          <a href="#video"class="nav__link">video</a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+
+  <main class="catalog">
+    <section class="catalog">
+      <div class="catalog__box">
+        <article class="catalog__card card" data-qa="card">
+          <img
+            class="card__image"
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27"
+          >
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__code">
+            Product code: 195434
+          </p>
+
+          <div class="card__reviews">
+            <ul class="stars stars--4">
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+            </ul>
+            Reviews: 5
+          </div>
+
+          <p class="card__price">
+            Price:
+            <span class="card__price-value">
+              $2,199
+            </span>
+          </p>
+
+          <a
+            class="card__button"
+            href="#"
+            data-qa="card-hover"
+          >
+          Buy
+          </a>
```

Comment:
Check indents for the content
```suggestion
          >
            Buy
          </a>
```